Automattic/wp-calypso
# Jetpack Cloud: update button styles

#### Changes proposed in this Pull Request

Note: This PR updates only a few colors in the `_jetpack-cloud.scss` to update the buttons and button-based components and to avoid changing the design of unrelated components (such as links, cards, etc) unexpectedly, but a thorough test is appreciated.

- Updates the colors in `_jetpack-cloud.scss`
- Updated the buttons styles in `client/jetpack-cloud/style.scss`

Context: dRt1pX85T68o5UNYK6AZro-fi-88%3A0 and p6TEKc-51M-p2

#### Testing instructions

- Check out this PR and run `yarn start-jetpack-cloud`.
- Go to http://jetpack.cloud.localhost:3000/
- Check the usage of the buttons and the screenshots below

#### Screenshots

These screenshots cover pretty much every possible button in Calypso.

Before | After
--|--
![image](https://user-images.githubusercontent.com/5714212/123706092-3729f280-d83e-11eb-849f-20653f892087.png)|![image](https://user-images.githubusercontent.com/5714212/123802520-c9270f00-d8c1-11eb-93ba-5f4747dd74a6.png)
